### PR TITLE
Create repo readme

### PR DESCRIPTION
**Pull request by chabroA**
Add basic Readme and contributing guidelines for the monorepo.

Both the Readme and Contributing guidelines will need to be improved and enhanced once the project is properly setup and structured.

This PR includes a [mermaid](https://mermaid-js.github.io/mermaid/#/) chart.

Cf. this [Github blog post](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/) to learn more about mermaid 🧜‍♀️ 

On VS Code, you can use [this](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid) extension to visualize it in VS Code's builtin markdown preview

TODO:

- [ ] Add list (badge) of each individual package version
- [ ] Add list (badge) of each individual package [licenses](https://choosealicense.com/licenses/) (or we can setup a common license at the monorepo level). Common one is [MIT](https://choosealicense.com/licenses/mit/)
- [ ] Enable and display CI (tests and building status) badge(s) once CI setup
- [ ] Add link to monorepo docusaurus website when available
- [ ] Link to reference implementations of client and server once available in _How does it works_ section
- [x] Add a visual explanation (chart / diagram) to illustrate connection / interaction between applications and wallets through API
- [ ] Develop the [_Contribution Guidelines_](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)
- [ ] Add a [_Code of conduct_](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project) (cf. [contributor-covenant](https://www.contributor-covenant.org/))
- [x] Add a [_Security policy_](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) -> https://github.com/LedgerHQ/platform-sdk/pull/4
- [ ] Add [_Support resources_](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,68 @@
-# Turborepo starter with pnpm
+<p align="center">
+ <img src="https://user-images.githubusercontent.com/9203826/154288895-670f5c23-81a1-4307-a080-1af83f7f8356.svg" align="center" alt="Ledger" />
+ <h2 align="center">Platform API</h2>
```
Comment (Justkant) [inline README.md:3]: Quick question, do we rename everything Wallet API ?

File: README.md [hunk 2]
```diff
@@ -1,42 +1,68 @@
...
-### Build
 
-To build all packages, run the following command:
+# How does it works
 
-```
-cd my-turborepo
-pnpm run build
-```
+The Platform API defines an interface for applications and wallets to interact within the Ledger ecosystem, allowing cross compatibilitiy between applications and wallets, in a currency agnostic way.
```
Comment (Justkant) [inline README.md:51]: We're also using platform api here while have wallet api in line 42

**Review by Justkant — COMMENTED** [review]
Quick question about the repo name, but I think we can start with this and improve it incrementally

File: README.md (hunk 1 above)
Comment (chabroA) [reply README.md:3]: I think we can rename everything Wallet API. It seems to be the de facto name of the project

**Comment by chabroA** [comment]
Recreated here https://github.com/LedgerHQ/platform-sdk/pull/5

**Pull request closed without merging**